apps/src/aichat/views/ChatWorkspace.tsx

```diff
@@ -168,22 +168,24 @@ const ChatWorkspace: React.FunctionComponent<ChatWorkspaceProps> = ({
         />
       )}
 
-      {canChatWithModel && (
-        <UserChatMessageEditor
-          editorContainerClassName={moduleStyles.messageEditorContainer}
-        />
-      )}
-      <div className={moduleStyles.buttonRow}>
-        <Button
-          text="Clear chat"
-          disabled={!canChatWithModel}
-          iconLeft={{iconName: 'eraser'}}
-          size="s"
-          type="secondary"
-          color="gray"
-          onClick={onClear}
-        />
-        <CopyButton />
+      <div className={moduleStyles.footer}>
+        {canChatWithModel && (
+          <UserChatMessageEditor
+            editorContainerClassName={moduleStyles.messageEditorContainer}
+          />
+        )}
+        <div className={moduleStyles.buttonRow}>
+          <Button
+            text="Clear chat"
+            disabled={!canChatWithModel}
+            iconLeft={{iconName: 'eraser'}}
```

Comment:
it's probably fine, but just noting - because it's an embedded anonymous object, this is gonna trigger a re-render of the `Button` component every time rendering falls into here. It's probably inconsequential, but at the same time if this part of the component gets called a lot it could add up.

This could be in either a `useMemo` or (even better) a hardwired constant outside of the component.

At the same time, it's admittedly almost assuredly a triviality, so I'm just noting to note it.

Reply:
Oh good catch, did not realize that! I'll create a constant (and I think this will be the case for other similar usages because the prop type of `iconLeft` is `FontAwesomeV6IconProps` which itself is an interface)